docs/notebooks/lda_training_tips.ipynb

```diff
@@ -0,0 +1,726 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# LDA: training tips\n",
+    "\n",
+    "LDA is a probabilistic hierarchical Bayesian model that is a mixture model as well as a mixed membership model... but we won't be getting into any of that.\n",
```

Comment:
True. The first sentence is also a tad snarky. I just removed the first sentence, does anything else need to be changed in that regard?